### ISSUE TITLE
Add hours and minutes to sitemap

### ISSUE THREAD

**Issue by krebil** [patrickdemooij9/SeoToolkit.Umbraco#216]
I recently talked to a search provider that uses the sitemap to figure out which pages to reindex. Having no hours and minutes on the time means that the search index can have stale content indefinitely when:

1. The search provider reindexes a page
2. You update the page the same day, but after the reindex has happened.

Now the timestamp won't change until the next time the page is changed, and so it will never be reindexed with the newest content.

From my understanding sitemaps can use any wc3 format: https://www.w3.org/TR/NOTE-datetime
So a complete date plus hours, minutes, and seconds could look like: YYYY-MM-DDThh:mm:ssTZD Where TZD is the time zone difference (example: 1997-07-16T19:20:30.45+01:00).

I think adding a configuration option that a can enable a more specific timestamp would be good, while keeping the change non-breaking.

If this is something that makes sense to you, I would be happy to provide a pull request.

**Comment by patrickdemooij9** [comment]
Hi @krebil 
Thanks for the great suggestion! Sorry, that it took a little bit for me to get back to you. I've implemented it in this commit: https://github.com/patrickdemooij9/SeoToolkit.Umbraco/commit/10e4e508fa0d78f541499f41d386d96cc90cdec8, so it will be included in the next release.

I've decided to adjust the default to the wc3 format (with time) as it make sense to also have that in the sitemap by default. If people don't agree, they are still able to revert it with a setting
